openshift.spec

```diff
@@ -47,8 +47,12 @@ Requires(postun): systemd
 %package node
 Summary:        OpenShift Node
 Requires:       %{name} = %{version}-%{release}
+%if 0%{?rhel} >= 7 || 0%{?fedora}
+Requires:       docker >= 1.6.2
+%else
```

Comment:
This shouldn't be necessary, we've been told that the docker packaging will always provide docker-io.